### ISSUE TITLE
NimbleOperatorRule

### ISSUE THREAD

**Issue by chrisamanse** [realm/SwiftLint#2215]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Bug Report

I'm trying to do TDD, and trying to add a test that will fail the `<` operator, and using `>` operator doesn't work since `<` operator always returns true initially. Thus, the assertion is now `expect(value).toNot(beLessThan(value2))`, where AFAIK, can't be rewritten using Nimble operator overloads.

```bash
$ swiftlint // nimble_operator enabled
warning: Nimble Operator Violation: Prefer Nimble operator overloads over free matcher functions. (nimble_operator)
```

```swift
// This triggers a violation:
expect(version_2_0_0).toNot(beLessThan(version_1_1_0))
```

### Environment

* SwiftLint version (run `swiftlint version` to be sure)? 0.25.1
* Installation method used (Homebrew, CocoaPods, building from source, etc)? Homebrew
* Paste your configuration file:

```yml
opt_in_rules:
        - nimble_operator
```

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)? No
* Which Xcode version are you using? Xcode 9.2
* Do you have a sample that shows the issue? Run `echo "expect(version_2_0_0).toNot(beLessThan(version_1_1_0))" | swiftlint lint --no-cache --use-stdin --enable-all-rules`

**Comment by marcelofabri** [comment]
`toNot(beLessThan(value2))` should be equivalent of `>= value2`.

**Comment by chrisamanse** [comment]
🤦‍♂️ totally forgot all about ranges. Thanks! Closing this issue..
